samples/drivers/video/capture_to_lvgl/boards/mini_stm32h743.conf

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_CLOCK_STM32_MCO1_SRC_HSI48=y
+CONFIG_CLOCK_STM32_MCO1_DIV=4
+
+CONFIG_LOG_BUFFER_SIZE=2048
+
+CONFIG_VIDEO_BUFFER_POOL_SZ_MAX=102400
+
+CONFIG_VIDEO_HFILP=y
+CONFIG_VIDEO_VFILP=y
```

Comment:
`FILP` -> `FLIP`

Reply:
Fixed and configuration moved to shield https://github.com/zephyrproject-rtos/zephyr/pull/77949.